homeassistant/components/pyload/sensor.py

```diff
@@ -124,12 +114,17 @@ class PyLoadSensor(SensorEntity):
     """Representation of a pyLoad sensor."""
 
     def __init__(
-        self, api: PyLoadAPI, entity_description: SensorEntityDescription, client_name
+        self,
+        api: PyLoadAPI,
+        entity_description: SensorEntityDescription,
+        client_name: str,
+        entry_id: str,
     ) -> None:
         """Initialize a new pyLoad sensor."""
         self._attr_name = f"{client_name} {entity_description.name}"
         self.type = entity_description.key
         self.api = api
+        self._attr_unique_id = f"{entry_id}_{entity_description.key}"
```

Comment:
Let's leave this for a follow up

Reply:
i should have known that you would say that 🙈